tools/submission/submission-checker.py

```diff
@@ -174,6 +174,9 @@
 MAX_ACCURACY_LOG_SIZE = 10 * 1024
 OFFLINE_MIN_SPQ = 24576
 TEST_DURATION_MS = 60000
+REQUIRED_COMP_PER_FILES = ["mlperf_log_summary.txt", "mlperf_log_detail.txt"]
```

Comment:
I think we need a command line option to disable the check in case audit submission comes in late.

Reply:
A flag to turn on/off checking on `compliance/` sounds like a good idea. @nv-rborkar maybe we can add it? Thanks

Reply:
Absolutely, pushing the change.

Reply:
@nv-rborkar Also, should we merge into r0.7 branch instead?

Reply:
Flag added. Please review & merge #735